src/chainparams.cpp

```diff
@@ -249,13 +252,95 @@ class CTestNetParams : public CChainParams {
     }
 };
 
+/**
+ * SigNet
```

Comment:
s/SigNet/Signet/

Reply:
A-ha! Thanks, my consistency is admirable... Fixed.